### PR TITLE
test(customd1): add `custom_d1/workload2` test

### PR DESCRIPTION
**Pull request by vponomaryov**
### Testing

- [ ]

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Comment by vponomaryov** [comment]
(PASSED) 2 simulated regions, 4 DB nodes in each region:
- [scylla-staging/valerii/vp-longevity-gce-custom-d1-workload2-hybrid-raid#9](https://argus.scylladb.com/test/7a8a0e29-e8e2-45c4-9271-ec040095e4a7/runs?additionalRuns[]=107862fd-4fcd-4899-91f3-e339a578adcb)

Setups with 6 and 7 nodes were failing with the `overload` problem on the `disrupt_nodetool_seed_decommission` nemesis.

File: vars/checkoutQaInternal.groovy [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 def call(Map params = [:]) {
     dir("scylla-qa-internal") {
-        git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
+        git(url: 'git@github.com:vponomaryov/scylla-qa-internal.git',
```
Comment (fruch) [inline vars/checkoutQaInternal.groovy:5]: please review this soe we can merge this one

**Review by fruch — DISMISSED** [review]
LGTM

File: vars/checkoutQaInternal.groovy (hunk 1 above)
Comment (vponomaryov) [reply vars/checkoutQaInternal.groovy:5]: I don't understand your request. Too many typos

Comment (vponomaryov) [reply vars/checkoutQaInternal.groovy]: Removed changes for this file assuming that the required changes will be merged in that repo.

**Pull request merged**